ipamd/ipamd.go

```diff
@@ -164,10 +169,24 @@ func (c *IPAMContext) decreaseIPPool() {
 	c.awsClient.FreeENI(eni)
 }
 
+func isAttachmentLimitExceededError(err error) bool {
+	return strings.Contains(err.Error(), "AttachmentLimitExceeded")
+}
+
 func (c *IPAMContext) increaseIPPool() {
+
+	if (c.maxENI > 0) && (c.maxENI <= c.dataStore.GetENIs()) {
```

Comment:
can c.maxENI be less than c.dataStore.GetENIs() ?

Reply:
c.maxENI can NOT be less than c.dataStore.GetENIs(), unless it is 0.

Reply:
That's what I thought. Changing the condition to '==' or at least adding a comment would avoid this confusion